devel/cluster/create-kind.sh

```diff
@@ -47,6 +47,9 @@ elif [[ "$K8S_VERSION" =~ 1\.22 ]] ; then
   KIND_IMAGE_SHA=$KIND_IMAGE_SHA_K8S_122
 elif [[ "$K8S_VERSION" =~ 1\.23 ]]; then
   KIND_IMAGE_SHA=$KIND_IMAGE_SHA_K8S_123
+elif [[ "$K8S_VERSION" =~ 1\.24 ]]; then
+  KIND_IMAGE_SHA=$KIND_IMAGE_SHA_K8S_124
+  KIND_IMAGE_REPO="eu.gcr.io/jetstack-build-infra-images/kind"
```

Comment:
It looks like an official image was pushed yesterday:
 https://hub.docker.com/r/kindest/node/tags

Reply:
nice! Will change to that

Reply:
.. however I cannot get the new image to work https://kubernetes.slack.com/archives/CEKK1KTN2/p1652186825534159

Reply:
Ah. Sorry for sending you down that rabbit hole. Perhaps they're still working on it.